libs/pages/summary/src/lib/ui/summary.tsx

```diff
@@ -1,62 +1,226 @@
-import { Box, Flex, LoadingOverlay, Paper, Table, Title } from '@mantine/core';
-import React from 'react';
+import { Box, Flex, Grid, LoadingOverlay, Paper, Title, Text, Center, Badge, Skeleton } from '@mantine/core';
 import { useTranslation } from 'react-i18next';
 import { BreadCrumbs } from '@yadoms/shared';
 import { loadSystemInformations } from '../summary-api';
 import { useQuery } from '@tanstack/react-query';
+import React, { ReactNode } from 'react';
+import { Container } from '@mantine/core';
+import semver from 'semver/preload';
+import classes from './summary.module.css';
+import {
+  DatabaseIcon, DatabaseVersionIcon,
+  LinuxLogo,
+  MacOSLogo,
+  PostgreSQLLogo, PowerIcon,
+  SQLIcon,
+  SQLiteLogo,
+  SystemIcon, VersionIcon,
+  WindowsLogo,
+  SizeIcon
+} from './icons';
 
-export function Summary() {
+
+
+const customization = {
+  Window: {color: "#4057dc", size:40},
+  Linux:{color: "#6f706e", size:40},
+  MacOs:{color: "#000000", size:40},
+  DefaultOs:{color: "#6e6e70", size:40},
+  SqLite:{color: "#66b2fc", size:40},
+  Postgres:{color: "#3177ff", size:40},
+  Database:{color: "#f14809", size:40},
+  DatabaseVersion: {color: "#cec483", size:40},
+  Power: {color: "#00ff28" ,size:40},
+  Version: {color: "#873be1", size:40},
+  Size: {color: "#e52121", size:40},
+  SQL: {color: "#0048fd", size:40},
```

Comment:
could u extract color values into an enum or named colored ( we could use them on other components)